include/openenclave/internal/tests.h

```diff
@@ -66,6 +66,12 @@ OE_EXTERNC_BEGIN
 
 #define OE_TEST_IGNORE(COND) OE_TEST_IF(COND, false)
 
+/* Multi-threaded testcases may need a specific base allocation
+ * when using allocators that make use of thread-local storage, such
+ * as tcmalloc or snmalloc.
+ */
```

Comment:
This seems like a useful shortcut to have for all the tests that make use of threads, because other allocators also have requirements that are typically not satisfied by the existing values (eg. tcmalloc wants 3 * 1024).

I think it would be right to tie this to USE_SNMALLOC, and to default to the largest existing hardcoded value otherwise, but I am very open to suggestions.